.prettierrc

```diff
@@ -1,3 +1,7 @@
 {
-  "singleQuote": true
+  "trailingComma": "es5",
```

Comment:
I would imagine that you could get away with `all` here.

Reply:
It will introduce 1k more problems.  And IMHO the problems aren't really problems - its just requiring a comma at the end of every list